cosmic-panel-bin/src/minimize.rs

```diff
@@ -63,7 +64,11 @@ pub fn set_rectangles(
         if old_info.surface == info.surface && old_info.rect != info.rect {
             old_info.rect = info.rect;
             changed = true;
-        } else if old_info.priority < info.priority || !old_info.surface.is_alive() {
+        } else if old_info.panel_type_priority < info.panel_type_priority
```

Comment:
Same as above, I think this would work, but it shouldn't need to compare the priorities here anyway.

Reply:
Fixed